pyQuARC/code/constants.py

```diff
@@ -33,7 +33,7 @@
         "providers",
         "science_keywords"
     ],
-    "xsd": [ DIF, "echo10_xml" ],
+    "xsd": [ f"{DIF}_xml", "echo10_xml" ],
```

Comment:
Use string interpolation for both or automate completely?

Reply:
298a2c3